www/js/services/sqlite_service.js

```diff
@@ -0,0 +1,100 @@
+(function(app) {
+
+  app.service('SQLiteService', function($q) {
+
+    var self;
+
+    var SQLiteService = function() {
+      self = this;
+      this.sqlPlugin = window.sqlitePlugin || window;
+      this.db = null;
+    };
+
+    SQLiteService.prototype.isDbEnable = function() {
+      return !!this.sqlPlugin;
+    };
+
+    SQLiteService.prototype.createDatabase = function() {
+      this.db = this.sqlPlugin.openDatabase(
+        window.rCreditsConfig.SQLiteDatabase.name,
+        window.rCreditsConfig.SQLiteDatabase.version,
+        window.rCreditsConfig.SQLiteDatabase.description, -1);
+    };
+    SQLiteService.prototype.ex = function() {
+      var txPromise = $q.defer();
+      txPromise.resolve(true);
+      return txPromise.promise;
+    };
+
+    SQLiteService.prototype.executeQuery_ = function(query, params) {
+      var txPromise = $q.defer();
+      var result;
+
+      this.db.transaction(function(tx) {
+        tx.executeSql(query, params, function(tx, res) {
+          console.log("executeSql OK: ", tx);
```

Comment:
We should clean up these consoles after we are done with the database stuff (probs not yet.)